src/wallet/coinselection.cpp

```diff
@@ -0,0 +1,370 @@
+#include "../tweak.h"
+#include "random.h"
```

Comment:
add coinselection.cpp and .h to .formatted-files

Reply:
cpp is in there, there isn't a .h.  Is there something in these 2 lines that looks incorrectly formatted?

Reply:
you're right, it is there...must have been github that made me think something was off with the formatting.